test/latestCode.test.ts

```diff
@@ -54,9 +54,9 @@ describe("latestCode", () => {
 
 describe("codeExists", () => {
   const bucket = "bucket"
-  const codeVersion = "2024-08-27T16:41:33.184Z"
-  const prefix = `serverless/webhook-handler/test/1549039293184-${codeVersion}/`
-  const prefixMatch = `serverless/webhook-handler/test/*-${codeVersion}/`
+  const codeVersion = "1549039293184-2024-08-27T16:41:33.184Z"
```

Comment:
Instead of just passing the date we will need to pass in the full key value. Both of these processes required looking s3 for these values so the process will not change. If this works we will add a ci job and updated readme in a follow up PR